01-introduction.Rmd

```diff
@@ -44,8 +44,12 @@ knitr::kable(x = d, caption = "Differences in emphasis between the fields of Geo
 ```
 
 While embracing recent developments in the field, we also wanted to pay respects to the wider field of Geography, with its 2000 history [@roller_eratosthenes_2010], and the narrower field of *Geographic Information System* (GIS) [@neteler_open_2008].
-Geography has played an important role in explaining and influencing humanity's relationship with the natural world, and this book aims to be a part of the 'Geographic tradition'.
+Geography has played an important role in explaining and influencing humanity's relationship with the natural world (just think of Humboldt's travels to South America), and this book aims to be a part of the 'Geographic tradition'.
```

Comment:
I think this needs a reference or web link (e.g. https://en.wikipedia.org/wiki/Alexander_von_Humboldt#Achievements_of_the_Latin_American_expedition ) and maybe some context.